.dependency_decisions.yml

```diff
@@ -0,0 +1,247 @@
+  - github.com/Masterminds/semver
+  - :who: 
+    :why: MIT
+    :versions: []
+    :when: 2020-05-18 21:53:07.976869000 Z
+- - :approve
+  - github.com/ghodss/yaml
+  - :who: 
+    :why: MIT
+    :versions: []
+    :when: 2020-05-18 21:53:23.703943000 Z
+- - :approve
+  - github.com/go-errors/errors
+  - :who: 
+    :why: MIT
+    :versions: []
+    :when: 2020-05-18 21:56:50.852312000 Z
+- - :approve
+  - github.com/go-redis/redis
+  - :who: 
+    :why: BSD-2-Clause
+    :versions: []
+    :when: 2020-05-18 21:57:42.588466000 Z
+- - :approve
+  - github.com/golang/mock
+  - :who: 
+    :why: Apache-2.0
+    :versions: []
+    :when: 2020-05-18 21:58:21.999176000 Z
+- - :approve
+  - github.com/googleapis/gnostic
+  - :who: 
+    :why: Apache-2.0
+    :versions: []
+    :when: 2020-05-18 21:58:54.936806000 Z
+- - :approve
+  - github.com/imdario/mergo
+  - :who: 
+    :why: BSD-3-Clause
+    :versions: []
+    :when: 2020-05-18 21:59:51.814750000 Z
+- - :approve
+  - github.com/mitchellh/mapstructure
+  - :who: 
+    :why: MIT
+    :versions: []
+    :when: 2020-05-18 22:00:27.006411000 Z
+- - :approve
+  - github.com/mitchellh/reflectwalk
+  - :who: 
+    :why: MIT
+    :versions: []
+    :when: 2020-05-18 22:00:51.549336000 Z
+- - :approve
+  - github.com/op/go-logging
+  - :who: 
+    :why: BSD-3-Clause
+    :versions: []
+    :when: 2020-05-18 22:01:22.807298000 Z
+- - :approve
+  - github.com/stretchr/testify
+  - :who: 
+    :why: MIT
+    :versions: []
+    :when: 2020-05-18 22:01:47.667117000 Z
+- - :approve
+  - golang.org/x/crypto
+  - :who: 
+    :why: BSD-3-Clause
+    :versions: []
+    :when: 2020-05-18 22:02:20.097249000 Z
+- - :approve
+  - golang.org/x/oauth2
+  - :who: 
+    :why: BSD-3-Clause
+    :versions: []
+    :when: 2020-05-18 22:02:46.458542000 Z
+- - :approve
+  - google.golang.org/api
+  - :who: 
+    :why: BSD-3-Clause
+    :versions: []
+    :when: 2020-05-18 22:03:20.443048000 Z
+- - :approve
+  - google.golang.org/genproto
+  - :who: 
+    :why: Apache-2.0
+    :versions: []
+    :when: 2020-05-18 22:04:32.937078000 Z
+- - :approve
+  - gopkg.in/h2non/gock.v1
+  - :who: 
+    :why: MIT
+    :versions: []
+    :when: 2020-05-18 22:05:15.887612000 Z
+- - :approve
+  - gopkg.in/src-d/go-git.v4
+  - :who: 
+    :why: Apache-2.0
+    :versions: []
+    :when: 2020-05-18 22:06:10.615852000 Z
+- - :approve
+  - gopkg.in/yaml.v2
+  - :who: 
+    :why: Apache-2.0
+    :versions: []
+    :when: 2020-05-18 22:06:41.946870000 Z
+- - :approve
+  - k8s.io/api
+  - :who: 
+    :why: Apache-2.0
+    :versions: []
+    :when: 2020-05-18 22:07:40.828327000 Z
+- - :approve
+  - k8s.io/apimachinery
+  - :who: 
+    :why: Apache-2.0
+    :versions: []
+    :when: 2020-05-18 22:08:28.944354000 Z
+- - :approve
+  - k8s.io/client-go
+  - :who: 
+    :why: Apache-2.0
+    :versions: []
+    :when: 2020-05-18 22:08:50.941564000 Z
+- - :approve
+  - github.com/logrusorgru/aurora
+  - :who: 
+    :why: Allowing MPL 2.0 for now
```

Comment:
is this because of the subdep? aurora itself seems to be DWTFYWTPL

https://github.com/logrusorgru/aurora/blob/master/LICENSE

Reply:
You are right. I fixed it.